src/exports.cc

```diff
@@ -798,6 +798,22 @@ Word wasi_unstable_args_sizes_get(void *raw_context, Word argc_ptr, Word argv_bu
   return 0; // __WASI_ESUCCESS
 }
 
+// __wasi_errno_t __wasi_clock_time_get(uint32_t id, uint64_t precision, uint64_t* time);
+Word wasi_unstable_clock_time_get(void *raw_context, Word clock_id, int64_t precision,
```

Comment:
`precision` should be `uint64_t`, not `int64_t`.

Reply:
fixed